### PR TITLE
migrated case property updates

### PR DESCRIPTION
**Pull request by nickpell**
Identified by Kriti and I in https://docs.google.com/spreadsheets/d/1qLTDb2K0x_8n_kUDy6t3-xRDiLw8uSMOJD90vmjnheE/edit#gid=0

@proteusvacuum @czue cc: @NoahCarnahan

### CODE REVIEW

File: custom/enikshay/nikshay_datamigration/models.py [hunk 1]
```diff
@@ -286,8 +286,8 @@ class Outcome(models.Model):
     @property
     def hiv_status(self):
         return {
-            None: None,
-            'NULL': None,
+            None: 'unknown',
```
Comment (czue) [inline custom/enikshay/nikshay_datamigration/models.py]: are there lots of these in the data? it's going to cause everyone to show up on the "HIV not updated" task lists, which is maybe fine/good.

**Review by czue — APPROVED** [review]
nice this looks great. glad we caught the treatment outcome thing before running this!

File: custom/enikshay/nikshay_datamigration/models.py (hunk 1 above)
Comment (proteusvacuum) [inline custom/enikshay/nikshay_datamigration/models.py]: When this is set to unknown or 'reactive', we need to make a drtb-hiv-referral case to the drtb-hiv location under the dto location.

Comment (nickpell) [inline custom/enikshay/nikshay_datamigration/models.py]: @proteusvacuum is this needed for 1.0?  Wondering if I can run the migration today and add the drtb-hiv-referral cases later or if that will break the 1.0 app.

Comment (proteusvacuum) [inline custom/enikshay/nikshay_datamigration/models.py]: I think we want this for 1.0

Comment (nickpell) [inline custom/enikshay/nikshay_datamigration/models.py]: https://github.com/dimagi/commcare-hq/pull/14770

Comment (nickpell) [inline custom/enikshay/nikshay_datamigration/models.py]: > are there lots of these in the data?

~16K

**Pull request merged**